extensions/core/interfaces/map/display.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div class="readonly-map">
+    <div class="interface-map-wrap">
+      <div class="map-display" id="directusMapDisplay" :style="{width:options.width+'px',height:options.height+'px'}">
+        <!-- Map Renders Here -->
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import mapMixin from "./map.js";
+
+export default {
+  name: "readonly-map",
+  mixins: [mapMixin],
+  data() {
+    return {
+      mapPlaceholder: "directusMapDisplay",
+      mode: "display"
+    };
+  },
+  watch: {
+    //? Do we need to re-render map when "value" changes from interface debugger?
+    value: function(newVal) {
+      this.viewMap(newVal);
+    }
+  },
+  mounted() {
+    this.viewMap(this.value);
+  },
+  methods: {
+    viewMap(latlng) {
+      this.latlng = JSON.parse(latlng);
```

Comment:
> JSON.parse(latlng)

The API has a JSON type too. If that's used, you don't have to / can't double parse the value